### PR TITLE
Us admin comic add request

### PR DESCRIPTION
**Pull request by vincentDubresson**

### CODE REVIEW

File: src/Controller/UserController.php [hunk 1]
```diff
@@ -42,6 +42,7 @@ public function list(): string
             }
         }
         $finalList = array_unique($finalList, SORT_REGULAR);
+        var_dump($finalList);
```
Comment (aldaitzwild) [inline src/Controller/UserController.php]: No we dont share our var_dump()

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

File: src/Controller/UserController.php (hunk 1 above)
Comment (vincentDubresson) [reply src/Controller/UserController.php]: I delete the var_dump in UserController.

**Review by aldaitzwild — CHANGES_REQUESTED** [review]
Conflicts with dev

File: src/View/Home/_header.html.twig [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
     
     <div class="position-absolute top-0 end-0 ">
-            <a herf="#" class="btn btn-lg" data-bs-toggle="modal" data-bs-target="#staticBackdrop"><img id="admin_icon" src="/assets/images/admin_icon.png"></a>
+            <a href="admin/list" class="btn btn-lg" {#data-bs-toggle="modal" data-bs-target="#staticBackdrop"#}><img id="admin_icon" src="/assets/images/admin_icon.png"></a>
```
Comment (aldaitzwild) [inline src/View/Home/_header.html.twig]: Still no dead code allowed

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

File: src/View/Home/_header.html.twig (hunk 2 above)
Comment (vincentDubresson) [reply src/View/Home/_header.html.twig]: I delete it.

**Review by aldaitzwild — APPROVED** [review]

**Pull request merged**